docs/sphinx/sections/api/apidocs/memoization.rst

```diff
@@ -1,8 +1,9 @@
-Versioning and Memoization (Experimental)
-=========================================
+Job-Level Versioning and Memoization (Deprecated)
+=================================================
 
-Dagster allows for code versioning and memoization of previous outputs based upon that versioning.
-Listed here are APIs related to versioning and memoization.
+Dagster has deprecated functionality that allows for job-level code versioning and memoization of previous op outputs based upon that versioning.
+
+This is currently deprecated in favor of asset versioning.
```

Comment:
Add a link to the docs about asset versioning here?

Reply:
I just tried this, but the links in RST don't appear to be rendering correctly: https://dagster-git-memoization-docs-elementl.vercel.app/_apidocs/memoization. Any tips?

Reply:
Yep! It took me a minute to figure this out, but here ya go:

```
# Syntax: `display text <https://somelink.com>`_

This is currently deprecated in favor of `asset versioning </concepts/assets/software-defined-assets#asset-code-versions>`_
```

It looks like reStructuredText doesn't use the flavor of Markdown we're used to, so styling is a bit different. I'm gonna add a formatting reference to the Dagster metadocs to make this info easily findable in the future.